runtime-modules/content/src/benchmarks/mod.rs

```diff
@@ -0,0 +1,541 @@
+        T::Origin::from(storage_wg_leader_signed),
+        None,
+        accepting_bags,
+        bucket_objs_size_limit,
+        bucket_objs_number_limit,
+    )
+    .unwrap();
+}
+
+fn create_distribution_buckets<T>(
+    lead_account_id: T::AccountId,
+    distribution_bucket_family_id: T::DistributionBucketFamilyId,
+    bucket_number: u32,
+) -> BTreeSet<DistributionBucketId<T>>
+where
+    T: Config + storage::Config,
+{
+    let storage_wg_leader_signed = RawOrigin::Signed(lead_account_id);
+    (0..bucket_number)
+        .map(|_| {
+            let distribution_bucket_index =
+                Storage::<T>::distribution_bucket_family_by_id(distribution_bucket_family_id)
+                    .next_distribution_bucket_index;
+
+            Storage::<T>::create_distribution_bucket(
+                T::Origin::from(storage_wg_leader_signed.clone()),
+                distribution_bucket_family_id,
+                true,
+            )
+            .unwrap();
+
+            DistributionBucketId::<T> {
+                distribution_bucket_family_id,
+                distribution_bucket_index,
+            }
+        })
+        .collect::<BTreeSet<_>>()
+}
+
+fn create_distribution_bucket_with_family<T>(
+    lead_account_id: T::AccountId,
+    bucket_number: u32,
+) -> (
+    T::DistributionBucketFamilyId,
+    BTreeSet<DistributionBucketId<T>>,
+)
+where
+    T: Config + storage::Config,
+{
+    let storage_wg_leader_signed = RawOrigin::Signed(lead_account_id.clone());
+
+    let db_family_id = Storage::<T>::next_distribution_bucket_family_id();
+
+    Storage::<T>::create_distribution_bucket_family(T::Origin::from(storage_wg_leader_signed))
+        .unwrap();
+
+    (
+        db_family_id,
+        create_distribution_buckets::<T>(lead_account_id, db_family_id, bucket_number),
+    )
+}
+
+pub fn create_data_object_candidates_helper(
+    starting_ipfs_index: u8,
+    number: u64,
+    size: u64,
+) -> Vec<DataObjectCreationParameters> {
+    let range = (starting_ipfs_index as u64)..((starting_ipfs_index as u64) + number);
+
+    range
+        .into_iter()
+        .map(|_| DataObjectCreationParameters {
+            size,
+            ipfs_content_id: vec![1u8],
+        })
+        .collect()
+}
+
+#[allow(clippy::too_many_arguments)]
+fn generate_channel_creation_params<T>(
+    dyn_bag_type: DynamicBagType,
+    storage_wg_lead_account_id: T::AccountId,
+    distribution_wg_lead_account_id: T::AccountId,
+    colaborator_num: u32,
+    storage_bucket_num: u32,
+    distribution_bucket_num: u32,
+    objects_num: u32,
+    max_bytes: u32,
+    max_obj_size: u64,
+) -> ChannelCreationParameters<T>
+where
+    T: Config
+        + storage::Config
+        + membership::Config
+        + balances::Config
+        + working_group::Config<StorageWorkingGroupInstance>
+        + working_group::Config<DistributionWorkingGroupInstance>,
+    T::AccountId: CreateAccountId,
+{
+    let chan_perm = CHANNEL_AGENT_PERMISSIONS.len() as u32 - 1;
```

Comment:
Why not just:
```suggestion
    let chan_perm = CHANNEL_AGENT_PERMISSIONS.len() as u32;
```
I think it makes sense for `get_permissions` function to take the length as a parameter (as opposed to index of the last element)

Reply:
I replaced by 
```rust
    let permissions = BTreeSet::from_iter(CHANNEL_AGENT_PERMISSIONS);
```
and removed the get_permissions